alexandria/oidc_auth/authentication.py

```diff
@@ -82,3 +83,33 @@ def cached_request(self, method, token, cache_prefix):
             func,
             timeout=self.OIDC_BEARER_TOKEN_REVALIDATION_TIME,
         )
+
+
+class DummyAuthenticationBackend(AlexandriaAuthenticationBackend):
```

Comment:
To be more inclusive, I suggest not using the term `dummy` but instead some alternative term. Maybe `MimicAuthenticationBackend()` could work?

Reply:
Renamed.

Reply:
There are many more instances of the term in this PR.

Reply:
Do you mean "inclusive" in the sense of: Dummies might be offended by this?

Reply:
@open-dynaMIX right - fixed them all now :)